### DISCUSSION TITLE
Cross System Container relationship from an extended workspace.

### DISCUSSION THREAD

**Discussion by eumpek** [structurizr/onpremises#143] [Q&A]
Hello everyone, 

We have our main workspace where we define certain systems and containers. On subsystems, we extend this main workspace and we are able to reuse systems and add relations. But, if we want to create a cross-system container-to-container relationship for systems that don't belong to the current extended workspace, it doesn't work. 

As per the examples shared [here](https://docs.structurizr.com/dsl/cookbook/container-view-multiple-software-systems/):

```dsl
workspace {
    model {
        s1 = softwareSystem "Software System 1" {
            c1 = container "Container 1"
        }

        s2 = softwareSystem "Software System 2" {
            c2 = container "Container 2"
        }
        
        c1 -> c2
    }

    views {
        container s1 {
            include c1 c2
            autoLayout lr
        }
    }
}
```

But what if I want to reference `s3` (that includes a single container `c3`)  when I extend the workspace, like:

```dsl
workspace extends s3.json {
    model {
        s1 = softwareSystem "Software System 1" {
            c1 = container "Container 1"
        }

        s2 = softwareSystem "Software System 2" {
            c2 = container "Container 2"
        }
        
        c1 -> c2

        c2 -> c3
    }

    views {
        container s1 {
            include c1 c2 c3
            autoLayout lr
        }
    }
}
```

then `c3` cannot be found. Any ideas on how to accomplish this?

**Comment by eumpek** [comment]
Hi @simonbrowndotje, do you maybe know if this is supported? Many thanks

**Reply by simonbrowndotje** [reply]
It's impossible to say without seeing the `s3.json` file and how you're creating it.
